app/decorators/course_decorator.rb

```diff
@@ -100,10 +139,10 @@ def has_site?(site)
   def funding_option
     if object.funding_type == "salary"
       "Salary"
-    elsif object.has_scholarship_and_bursary?
+    elsif has_scholarship_and_bursary?
       "Scholarship, bursary or student finance if you’re eligible"
```

Comment:
I understand that this is existing code, but is it correct? It seems quite possible that courses will only have one or the other, so:

Q1. should only courses that have both behave this text?
Q2. is "student finance" the same as "early_career_payments"? Just trying to match the wording here to what's in the DB, where we have `bursary_amount` and `scholarship` and `early_career_payments`, but no "student finance".
Q3. should we move this to locales?

Reply:
Q2: no they not the same thing